File: Python Folder/PRAK305-2210817310011-M.Daffa_Az-Zikra.py
ss = int(input())
if ss<60 and ss>=0:
    print('00:00:{:02d}'.format(ss))
elif ss>=60 and ss<3600:
    print('00:{:02d}:{:02d}'.format(int(ss/60),int(ss%60)))
elif ss>=3600 and ss<86400:
    print('{:02d}:{:02d}:{:02d}'.format(int(ss/3600),int((ss%3600)/60),int((ss%3600)%60)))
elif ss>=86400:
    print('{} hari {:02d}:{:02d}:{:02d}'.format(int(ss/86400),int((ss%86400)/3600),int(((ss%86400)%3600)/60),int(((ss%86400)%3600)%60)))
else:
    print('Error',complex(ss))

File: C Folder/PRAK305-2210817310011-M.DaffA_Az-Zikra.c
#include<stdio.h>
int main(){
    int ss;
    scanf("%d",&ss);
    (ss<60 && ss>=0) ? printf("00:00:%02d",ss)
    : (ss>=60 && ss<3600) ? printf("00:%02d:%02d",ss/60,ss%60)
    : (ss>=3600 && ss<86400) ? printf("%02d:%02d:%02d",ss/3600,(ss%3600)/60,(ss%3600)%60)
    : (ss>=86400) ? printf("%d hari %02d:%02d:%02d",ss/86400,(ss%86400)/3600,((ss%86400)%3600)/60,((ss%86400)%3600)%60)
    : printf("Error");
    return 0;
}